src/config/networks/network.d.ts

```diff
@@ -0,0 +1,59 @@
+import { ETHEREUM_NETWORK } from 'src/logic/wallets/getWeb3'
+
+type DisplayFlag = 'enable' | 'disable'
+
+// matches src/logic/tokens/store/model/token.ts `TokenProps` type
+type Token = {
+  address: string
+  name: string
+  symbol: string
+  decimals: number
+  logoUri?: string
+}
+
+type NetworkSettings = {
+  id: ETHEREUM_NETWORK,
+  color: string,
+  label: string,
+  nativeCoin: Token,
+}
+
+// something around this to display or not some critical sections in the app, depending on the network support
+// I listed the ones that may conflict with the network.
+// If non is present, all the sections are available.
+type SafeFeatures = {
+  safeApps?: DisplayFlag,
+  collectibles?: DisplayFlag,
+  contractInteraction?: DisplayFlag
+}
+
+type GasPrice = {
+  gasPrice: number
```

Comment:
I'm not sure about this, I thought it was going to be a URL or NULL, if a URL is provided, then the service in charge to resolve the gas calculation will do all it's needed to return a value. If it's NULL the service will return a hardcoded value. 

If you don't like that alternative and would suggest typing this value like so:
`gasPriceInfo: number | string`

Reply:
mhh... but, where will you define the hardcoded value depending on the network you're using?

In regard to the type, why will you need a `string` type for that value?

Reply:
I don't see the problem here. This may be necessary to be different depending on the network, so the fallback value should be configurable

Any more feedback to add @nicosampler ?

Reply:
no, that's fine